apidoc/Snapshotter.yml

```diff
@@ -0,0 +1,132 @@
+---
+name: Modules.Map.Snapshotter
+summary: Snapshotter is used to allow screen shots to be taken of a specified region or a mapview.
+
+extends: Titanium.UI.View
+excludes:
+    events: [ 'click', singletap', 'doubletap', 'dblclick', 'longpress', 'pinch',
+              'swipe', 'touchstart', 'touchend', 'touchcancel', 'touchmove', 'twofingertap']
+
+since: {iphone: "6.0.0", ipad: "6.0.0" }
+platforms: [iphone, ipad]
+methods:
+
+  - name: takeSnap
+    summary: Takes a snap shot of of the map corresponding to the region property or a snap of the mapView if
+        passed in as a second parameter.
+    returns:
+        type: Titanium.Blob
+    parameters:
+      - name: callback
+        summary: | 
+            Function to be invoked upon completion. If non-null, this method will be performed 
+            asynchronously. If null, it will be performed immediately.
+        type: Callback<Titanium.Blob>
+        optional: false
+      - name: map
+        summary: Map View to be captured by the snap shot.
+        type: view<Modules.Map.View>
+        optional: true
+
+  
+properties:
+  - name: region
+    summary: |
+        A dictionary specifying the location and zoom level of the map.
+    type: MapRegionTypev2<Modules.Map.View.MapRegionTypev2>
+    default: latitude 0, longitude 0
+
+  - name: showsPointsOfInterest
+    summary: When this property is set to YES, the map displays icons and labels for 
+        restaurants, schools, and other relevant points of interest.
+    type: Boolean
+    default: true
+
+  - name: mapType
+    summary: |
+        Map type constant, either <Modules.Map.NORMAL_TYPE>, <Modules.Map.SATELLITE_TYPE>,
+        <Modules.Map.TERRAIN_TYPE>, <Modules.Map.HYBRID_TYPE>, <Modules.Map.HYBRID_FLYOVER_TYPE> or 
+        <Modules.Map.SATELLITE_FLYOVER_TYPE>.
+    type: Number
+    default: NORMAL_TYPE
+
+  - name: showsBuildings
+    summary: Determines whether building will be shown on the map. The mapType property must be set 
+        to <Modules.Map.STANDARD_TYPE> for extruded buildings to be displayed.
+    type: Boolean
+    default: true
+
+ examples:
+  - title: Snapshot without a mapView
+    example: |
+       
+                var MapModule = require('ti.map');
+
+                var win = Ti.UI.createWindow({
+                    backgroundColor: 'white'
+                });
+
+
+                var Snapshotter = MapModule.createSnapshotter({
+                    mapType: MapModule.HYBRID_TYPE,
+                    region: {
+                        latitude: 37.3382,
+                        longitude: -121.8863,
+                        latitudeDelta: 0.4,
+                        longitudeDelta: 0.4
+                    }
+                });
+
+
+                Snapshotter.takeSnap(function(e) {
+                    var imageView = Ti.UI.createImageView({
+                        image: e.image,
+                        width:Ti.UI.FIlL,
+                        height:Ti.UI.FILL
```

Comment:
Missing spaces before property-assignment, also typo in `Ti.UI.FIlL` -> `Ti.UI.FILL`. Did you test all examples?